--- squashfs-root/resources/app/src/engine/license-manager.js	2020-07-27 18:02:42.156232792 -0400
+++ squashfs-root-modified/resources/app/src/engine/license-manager.js	2020-07-27 18:05:14.048895257 -0400
@@ -126,8 +126,8 @@
     this.validate().then(() => {
       setStatus(this, true)
     }, () => {
-      setStatus(this, false)
-      UnregisteredDialog.showDialog()
+      setStatus(this, true)
+      // UnregisteredDialog.showDialog()
     })
   }
 
